documentation/book/proc-deploying-cluster-operator-kubernetes-to-watch-whole-cluster.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+// assembly-operators-cluster-operator.adoc
+
+[id='deploying-cluster-operator-kubernetes-to-watch-whole-cluster-{context}']
+= Deploying the Cluster Operator to watch the whole cluster
```

Comment:
Maybe it's useful using `... whole {ProductPlatformName} cluster` in the title as well?

Reply:
Yes, but IMHO the context makes it quite clear. And if not it is explained in the sentence just below it.